storage-resize-images/extension.yaml

```
@@ -168,6 +168,36 @@ params:
     example: thumbnails
     required: false
 
+  - param: ABSOLUTE_PATH_LIST
+    label: List of absolute paths for resized images
+    description: >
+      A comma-separated list of absolute paths to restrict the 
+      images to be resized. For example, to only resize the images 
+      stored in `/users/pictures` directory, specify the path `/users/pictures`. 
+      If you prefer to resize every image uploaded to the storage bucket, 
+      leave this field empty.
+    type: string
+    example: "/users/avatars,/design/pictures"
```

Let's modify this param and its label/description so that its purpose is more clear. Here's my attempt:

```suggestion
  - param: INCLUDE_PATH_LIST
    label: Specific paths that include images to resize
    description: >
      Restrict storage-resize-images to only resize specific areas of your Storage bucket by 
      supplying a comma-separated list of absolute paths. For example, to only resize the images 
      stored in `/users/pictures` directory, specify the path `/users/pictures`. 
      If you prefer to resize every image uploaded to the storage bucket, 
      leave this field empty.
    type: string
    example: "/users/avatars,/design/pictures"
```

@kevinthecheung any thoughts on how to phrase this?

---

Once we decide on wording, let's check if the same edits need to be applied to `EXCLUDE_PATH_LIST`

---

We should also note somewhere that, no matter what paths are set here, the extension will still trigger for every upload to a bucket. It will just exit itself if it notices that it was triggered on a file it isn't supposed to resize

---

I like your suggestion about to reword the option `ABSOLUTE_PATH_LIST` to `INCLUDE_PATH_LIST`, it sounds better aligned to the sense of the config option itself. So it will be `include / exclude` :+1:

---

How about:
```
 - param: INCLUDE_PATH_LIST
    label: Paths that contain images you want to resize
    description: >
      Restrict storage-resize-images to only resize images in specific locations in your Storage bucket by 
      supplying a comma-separated list of absolute paths. For example, to only resize the images 
      stored in `/users/pictures` directory, specify the path `/users/pictures`. 
      If you prefer to resize every image uploaded to the storage bucket, 
      leave this field empty.
    type: string
    example: "/users/avatars,/design/pictures"
```

---

Looks like Kevin's suggestion has been added. Closing out this thread.